code/ttt-reborn.csproj

```diff
@@ -3,8 +3,8 @@
 	<PropertyGroup>
 		<TargetFramework>net5.0</TargetFramework>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
-		<AssemblyName>tttreborn</AssemblyName>
-		<PackageId>tttreborn</PackageId>
+		<AssemblyName>ttt-reborn</AssemblyName>
+		<PackageId>ttt-reborn</PackageId>
```

Comment:
why is this changed back? Didn't Alf just change it from `ttt-reborn` to `tttreborn`

Reply:
We decided on marking any .csproj files on the gitignore since they are unique to each machine.

From my understanding of gitignore it'll start ignoring files once it's commited to repo. So will delete file once merged.

Reply:
oh, alright. I missed it, sorry!